Formula/rubyfmt.rb

```diff
@@ -1,8 +1,10 @@
 class Rubyfmt < Formula
   desc "Ruby autoformatter"
   homepage "https://github.com/penelopezone/rubyfmt"
-  url "https://github.com/penelopezone/rubyfmt/archive/v0.2.0.tar.gz"
-  sha256 "68ebc0fd30933b1e27b609cc34c69a3bc886747d11c5b949e460ce01814adaeb"
+  url "https://github.com/penelopezone/rubyfmt.git",
+    branch:   "trunk",
+    tag:      "v0.8.0",
```

Comment:
You don't need both `branch` and `tag`, I'd say `tag` is relevant here for the stable download. I notice that you did lose the `head` portion though, you probably want to keep `branch: "trunk"` there.